src/reporting/file-issue-for-review.js

```diff
@@ -177,13 +193,17 @@ if (require.main === module) {
 	      }
 	    } catch (err) {
 	      console.error(`Failed to commit error report for ${specResult.title}`, err);
-	      fs.unlink(issueFilename);
+	      await fs.unlink(issueFilename);
 	      execSync(`git checkout ${currentBranch}`);
 	    }
 	  }
 	}
       }
     }));
+    if (nolongerRelevantReports.size) {
+      console.log("The following reports are no longer relevant, deleting them", [...nolongerRelevantReports]);
+      [...nolongerRelevantReports].forEach(async issueFilename =>  await fs.unlink(issueFilename));
```

Comment:
The `forEach` call won't wait for `unlink` calls to be done.

```suggestion
      for (const issueFilemane of nolongerRelevantReports) {
        await fs.unlink(issueFilename);
      }
```